### DISCUSSION TITLE
Octoprint printer / CNC same PI - Swapping PI to talk to CNC Octo still trying to write to printer LCD

### DISCUSSION THREAD

**Discussion by garyzz** [synman/Octoprint-Bettergrblsupport#96] [Ideas]
When running a CNC job my jobs periodically stall and timeout. My friend helped me track it down to what might be caused by messages sent to LCD screen (which I do not have coupled to the Pi when running CNC).   These messages may be caused by another plugin that is attempting to send state to the LCD screen of my printer.   Any suggestions on how to surpress these LCD messages from interfering when running a job in Better GRBL?

**Comment by synman** [comment]
I'll need to see your terminal log to make heads or tails of it.

We'll likely be able to track down the source and see about disabling them or adding an ignore function to the plug-in.

I can basically eat them if I know what to trap for.  If they are useful I can see about rerouting them to the notification panel in Octoprint too.

Shell

**Comment by synman** [comment]
I should advise you. I do not support running a dual setup on a single pi.

There is far too much at play here.

I highly recommend you use one pi for your 3d printer and another one for your CNC/laser.

**Reply by garyzz** [reply]
Understood. Thanks and I will follow your recommendation. Good. Cheers.
